site/content/en/docs/contrib/releasing/binaries.md

```diff
@@ -12,6 +12,11 @@ description: >
 * Two minikube repos checked out locally:
   * Your personal fork
   * Upstream  
+  
+## Update the Kubernetes version
+
+* Run `make update-kubernetes-version` from your local upstream repo copy
+* If any files are updated, combine these changes with the Release Notes PR 
```

Comment:
this would need to be a separate PR. so we can test the health of the new kubernets version

Reply:
fixed